tasks/template_compose.yml

```diff
@@ -1,16 +1,16 @@
 - name: Ensure path to service folder for {{ service }}
-  file:
-    path: "{{ project_dir }}/{{service}}"
+  ansible.builtin.file:
+    path: "{{ project_dir }}/{{ service }}"
     state: directory
     owner: "{{ ansible_facts.user_id }}"
     group: "{{ ansible_facts.user_id }}"
     mode: '0755'
 
 - name: Template compose file for {{ service }}
-  template:
-    src: ../services/{{ service }}/templates/compose.yml.j2
+  ansible.builtin.template:
+    src: ../services/{{ service }}/compose.yml.j2
```

Comment:
why change the source file here ? wouldn't this break templating the compose files ?